portfolio/src/main/webapp/script.js

```diff
@@ -13,28 +13,55 @@
 // limitations under the License.
 
 
-/**
- * @return {string} 
- * a random greeting
+/* @return {string}
+ * Generate a random greeting
  */
 function generateRandomGreeting() {
     //TODO use google translated to get greetings in different languages
     const greetings =
         ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
     // Pick a random greeting.
-    var greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    const greeting = greetings[Math.floor(Math.random() * greetings.length)];
     return greeting;
 }
 
+/* Retrieve user comments and display them */
+async function getUserComments(){
+    try{
+    const response = await fetch("/data");
+    const data = await response.json();
+
+    const commentEl = document.querySelector("#user-comments");
+        if(typeof(commentEl) != 'undefined' && commentEl != null){
```

Comment:
optional nit: if you find that the entire function is inside an if statement, sometimes it can help make code look cleaner by returning early, so that the functionality isnt nested in the if 
So in this case, you could do:

if (typeof(commentEl) == 'undefined' || commentEl == null {
   return; // break out of the function early
}
for(const comment in data) ..... // rest of your code